### ISSUE TITLE
img2img inter-images resolution wrong

### ISSUE THREAD

**Issue by JackeyDeng** [AlUlkesh/sd_save_intermediate_images#35]
Hello, I am using StableDiffusion WebUI to do 1X SR, my input image is 512*512,  but when I save intermediate images, its resolution is 128*128, is my settings wrong?
![2023-08-31 18-10-14 的屏幕截图](https://github.com/AlUlkesh/sd_save_intermediate_images/assets/34648289/a00538aa-ad4c-4c61-a47e-ea1a47848224)

**Comment by AlUlkesh** [comment]
Yes, the reason is probably what this remark in the readme refers to:

> Please be aware that Image creation progress preview mode in the webui's settings affects how and especially at what size the intermediate images are created.

Try setting your preview mode to "Full".

**Comment by JackeyDeng** [comment]
Thanks, it works if I set the preview mode to "Full"
![2023-09-01 15-19-38 的屏幕截图](https://github.com/AlUlkesh/sd_save_intermediate_images/assets/34648289/62a1f0b9-3714-40dc-ad15-f8f7fdab22a0)
But it does not support DDIM, which file should I modify to support DDIM? I think its almost the same to get DDIM intermediate images, just to decode the intermediate latents?

**Comment by AlUlkesh** [comment]
When I made that extension, that just wasn't possible with the a1111 infrastructure. You could only get that information from the kdiffusion samplers.

However, I just saw this in the new a1111 release notes:

> rework DDIM, PLMS, UniPC to use CFG denoiser same as in k-diffusion samplers

So, perhaps it is possible now. I'll have to look into it.
